clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2157,6 +2157,16 @@ void CodeGenModule::SetCommonAttributes(GlobalDecl GD, llvm::GlobalValue *GV) {
         VD->getStorageDuration() == SD_Static)
       addUsedOrCompilerUsedGlobal(GV);
   }
+
+  if (getLangOpts().SYCLIsDevice) {
```

Comment:
Changes look good to me. Could you please add a comment for future reference? Thanks

Reply:
Sure, done.